### ISSUE TITLE
feature Doesn't empty `src` in `g-image`

### ISSUE THREAD

**Issue by tyankatsu0105** [gridsome/eslint-plugin-gridsome#4]
**Please describe what the rule should do:**

**What category should the rule belong to?**

- [ ] Enforces code style
- [x] Warns about a potential error
- [ ] Suggests an alternate way of doing something
- [ ] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**
```vue
<template>
  <g-image
  class="foo"
  width="300"
  />
</template>
```

This would be correct:
```vue
<template>
  <g-image
  :src="foo.bar.src"
  class="foo"
  width="300"
  />
</template>
```

```vue
<template>
  <g-image
  src="/path/foo.jpg"
  class="foo"
  width="300"
  />
</template>
```

**Additional context**
